api/src/test/java/org/openmrs/util/DatabaseUpdaterTest.java

```diff
@@ -29,15 +31,42 @@
 	 * @see DatabaseUpdater#updatesRequired()
 	 */
 	@Test
-	public void updatesRequired_shouldAlwaysHaveAValidUpdateToLatestFile() throws LockException {
+	public void updatesRequired_shouldAlwaysHaveAValidUpdateToLatestFile() throws Exception {
 		// expects /metadata/model to be on the classpath so that
 		// the liquibase-update-to-latest.xml can be found.
 		try {
 			DatabaseUpdater.updatesRequired();
 		}
-		catch (RuntimeException rex) {
-			log.error("Runtime Exception in test for Validation Errors");
+		catch (Exception ex) {
+			log.error("Exception in test for Validation Errors");
 		}
 		// does not run DatabaseUpdater.update() because hsqldb doesn't like single quotes in strings
 	}
+	
+	@Test(expected = IllegalArgumentException.class)
+	public void shouldRejectNullAsChangelog() throws DatabaseUpdateException, InputRequiredException {
+		DatabaseUpdater.executeChangelog(null, (DatabaseUpdater.ChangeSetExecutorCallback) null);
+	}
+	
+	@Test
+	public void shouldRejectNullAsChangelogFilenames() {
+		try {
+			DatabaseUpdater.getUnrunDatabaseChanges((String[]) null);
+			fail();
+		}
+		catch (RuntimeException re) {
+			assertTrue(re.getCause() instanceof IllegalArgumentException);
+		}
+	}
+	
+	@Test
+	public void shouldEmptyArrayAsChangelogFilenames() {
```

Comment:
Did you miss a word in the above method name?

Reply:
Indeed, thanks for spotting this.

Reply:
And done.